security/compliance_operator/co-management/compliance-operator-installation.adoc

```diff
@@ -8,6 +8,12 @@ toc::[]
 
 Before you can use the Compliance Operator, you must ensure it is deployed in the cluster.
 
+[NOTE]
```

Comment:
I don't think there is a style guide rule about this anywhere, but do you think maybe the NOTE should be placed under the two IMPORTANT admonitions? Technically, IMPORTANT would be considered higher severity and arguably should be read first.

Up to you though